pkg/hostagent/hostagent.go

```diff
@@ -357,6 +358,146 @@ func (a *HostAgent) processGuestAgentEvents(ctx context.Context, localUnix strin
 	return io.EOF
 }
 
+func forwardTCP(ctx context.Context, sshConfig *ssh.SSHConfig, port int, local, remote string, cancel bool) error {
+	// on macOS, listening on 127.0.0.1:80 requires root while 0.0.0.0:80 does not require root.
+	// https://twitter.com/_AkihiroSuda_/status/1403403845842075648
+	// We use "pseudoloopback" forwarder that listens on 0.0.0.0:80 but rejects connections from non-loopback src IP.
+	if runtime.GOOS == "darwin" {
```

Comment:
This is a very long "true" branch, with a very short "else" branch a long time later. Maybe change to

```
if runtime.GOOS != "darwin" {
    return forwardSSH(ctx, sshConfig, port, local, remote, cancel)
}
```

and then outdent the rest of the function by one level (don't use an "else" branch)?

Reply:
Moved to `port_darwin.go`